proposals/0003-pr-process.md

```diff
@@ -0,0 +1,22 @@
+---
+author: Tikhon Jelvis
+date-accepted: N/A
+discussion: https://github.com/haskell-org/committee/pull/2
+---
+
+# Accepting Website PRs
+
+To quickly triage and accept PRs to the website, we need a process for reviewing, merging and deploying changes to the website. We adopted a process for this before we had this proposal repo, and the process helped us respond to and merge PRs faster than before. This proposal codifies that process to help make our policies explicit and transparent.
+
+Guidelines for accepting PRs to the website:
+
+  1. Purely technical fixes—like fixing the `default.nix` file, broken links, typos and HTML glitches—require one member of the committee to approve.
```

Comment:
Should we waive the review requirement for maintainers in the case of an immediate breakage? Maybe we can add a line like:

For urgent issues impacting the security or availability of the site, maintainers may push a minimal fix without a review. Members of the committee should review any changes that were pushed without review as soon as possible to ensure a revert or follow up isn’t needed.

Reply:
For this one, let's keep the requirement as-is, and maybe also provide some way to ping the whole committee on Slack or something if we need a fast response.

If this requirement starts getting in the way, we can revisit it down the line.